### PR TITLE
Compilation and test fixes

### PR DESCRIPTION
**Pull request by tangrammer**
- [FIX: Use of undeclared Var com.kubelt.lib.error/conform*](https://github.com/kubelt/kubelt/runs/6600348899?check_suite_focus=true#step:8:379)

```
------ WARNING #1 - :undeclared-var --------------------------------------------
 File: /home/runner/work/kubelt/kubelt/src/main/com/kubelt/rpc/schema.cljc:30:5
--------------------------------------------------------------------------------
  27 |      (schema client prefix schema-doc defaults)))
  28 | 
  29 |   ([client prefix schema-doc options]
  30 |    (lib.error/conform*
-----------^--------------------------------------------------------------------
 Use of undeclared Var com.kubelt.lib.error/conform*
--------------------------------------------------------------------------------
  .....
```

- [Fix node test output error](https://github.com/kubelt/kubelt/pull/426/commits/10c0ce563bc2dc4ca54a2f92c65452d6f983a83c) ====> [expected failing build](https://github.com/kubelt/kubelt/runs/6609836975?check_suite_focus=true#step:8:399)
- [Fix schema-test](https://github.com/kubelt/kubelt/pull/426/commits/5017acbedc845509e30249946bfbfffb10d1e499)

### CODE REVIEW

File: bb.edn [hunk 1]
```diff
@@ -169,7 +169,8 @@
 
   test:sdk:cljs:develop
   {:doc "Build and run development version of sdk tests"
-   :task (shell "npx shadow-cljs compile sdk-test")}
+   :task (do (shell "npx shadow-cljs compile sdk-test ")
```
Comment (Cosmin-Parvulescu) [inline bb.edn:172]: I'm not sure what this does :)

File: shadow-cljs.edn [hunk 2]
```diff
@@ -170,7 +170,7 @@
    :compiler-options {:reader-features #{:node}}
    :output-to "target/sdk-test.js"
    :ns-regexp "(lib|sdk|com\\.kubelt\\.rpc)\\..*-test$"
-   :autorun true}
+   :autorun false}
```
Comment (Cosmin-Parvulescu) [inline shadow-cljs.edn:173]: Or what the effects of stopping autorun could be

**Review by Cosmin-Parvulescu — COMMENTED** [review]
The schema_test LGTM 🚀 The rest I don't understand all that well though it looks like it fixes the test error.

File: bb.edn (hunk 1 above)
Comment (tangrammer) [reply bb.edn:172]: it runs 2 shell commands

File: shadow-cljs.edn (hunk 2 above)
Comment (Cosmin-Parvulescu) [reply shadow-cljs.edn:173]: Was that our rogue ```nil``` value? :P

File: bb.edn (hunk 1 above)
Comment (crimeminister) [reply bb.edn:172]: Is there any value in splitting these into separate tasks and we use bb's dependency mechanism to run them in sequence?

File: shadow-cljs.edn (hunk 2 above)
Comment (crimeminister) [reply shadow-cljs.edn:173]: Anyone who was relying on the tests running immediately after compilation will instead have to run them manually. If that's a bother we can sequence the compilation and the execution of the tests using `bb`.

**Review by crimeminister — DISMISSED** [review]

**Review by crimeminister — APPROVED** [review]

**Pull request merged**